docs/source/unsloth_integration.md

```diff
@@ -1,7 +1,144 @@
-# Unsloth Integration
+# Unsloth Integration Guide
 
-<Tip warning={true}>
+Unsloth is a lightweight library designed to accelerate and optimize the training of large language models (LLMs) while reducing memory usage. This guide provides step-by-step instructions for integrating Unsloth into your existing workflows.
 
-Section under construction. Feel free to contribute!
+## Table of Contents
+- [Installation](#installation)
```

Comment:
No need for a toc, you will have one automatically on the right with the doc builder